packages/botonic-react/src/components/index-types.ts

```diff
@@ -151,7 +151,9 @@ export interface ThemeProps extends StyleProp {
     enableArrows?: boolean
   }
   reply?: StyleProp & CustomProp
-  triggerButton?: ImageProp & StyleProp & CustomProp
+  triggerButton?: { notificationsEnabled?: boolean } & ImageProp &
```

Comment:
Suggestion: Why not adding a `TriggerButtonProps` with all these types?

Reply:
for now I do not apply it but I will take it into account later. Thanks Adria